addons/test_mass_mailing/tests/test_mailing_statistics_sms.py

```diff
@@ -43,13 +68,36 @@ def test_mailing_statistics_sms(self):
         for record_idx in (0, 2, 3):
             self.gateway_sms_click(mailing, target_records[record_idx])
 
-        # check mailing statistics
+        for record_idx in [4, 5, 6]:
```

Comment:
this is still tested on a sms mailing no ?

![image](https://github.com/odoo/odoo/assets/54316226/d1d2d93f-558d-48c6-8779-f5228d39a74f)